experimenter/experimenter/experiments/tests/test_models.py

```diff
@@ -823,16 +823,49 @@ def test_start_date_returns_None_for_not_started_experiment(self):
         )
         self.assertIsNone(experiment.start_date)
 
+    def test_proposed_release_date_returns_None_for_not_started_experiment(self):
+        experiment = NimbusExperimentFactory.create_with_lifecycle(
+            NimbusExperimentFactory.Lifecycles.CREATED,
+        )
+        self.assertIsNone(experiment.proposed_release_date)
+
     def test_end_date_returns_None_for_not_ended_experiment(self):
         experiment = NimbusExperimentFactory.create_with_lifecycle(
             NimbusExperimentFactory.Lifecycles.CREATED,
         )
         self.assertIsNone(experiment.end_date)
 
-    def test_launch_month_returns_month_for_started_experiment(self):
+    def test_launch_month_returns_release_date_month_for_started_first_run_experiment(
+        self,
+    ):
+        experiment = NimbusExperimentFactory.create_with_lifecycle(
+            NimbusExperimentFactory.Lifecycles.LIVE_ENROLLING,
+            proposed_release_date=datetime.date.today() + datetime.timedelta(days=1),
+            is_first_run=True,
+        )
+
+        assert experiment.proposed_release_date
```

Comment:
why this assert? 🤔

Reply:
These tests are all copy/pasted from the existing tests to extend to testing `proposed_release_date` in addition to `start_date` :)

Reply:
I think we should assert with some value as expected value